deploy.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.9'
+
+services:
+
+  db:
+    image: postgres:14.1
+    environment:
+      POSTGRES_USER: bugle_prod
+      POSTGRES_DB: bugle_prod
+      POSTGRES_PASSWORD_FILE: /run/secrets/db_password
```

Comment:
question: How does this path work to retrieve the db password?

Reply:
`docker-compose` automatically [creates a runtime bind mount](https://docs.docker.com/engine/swarm/secrets/#use-secrets-in-compose) from the secrets file to the `/run/secrets/<secret-name>` path.